src/components/EditableTextarea/EditableTextarea.tsx

```diff
@@ -214,14 +223,27 @@ export class EditableTextarea extends React.PureComponent<
             value: [item],
             event: e,
           })
-          this.textArea.current.blur()
         }
       )
     }
   }
 
+  handleOnKeyUp = e => {
+    const code = e.key
+
+    /* istanbul ignore next */
+    const stop = () => e.preventDefault() && e.stopPropagation()
+
+    // Escape route tested
+    /* istanbul ignore if */
+    if (code === key.ESCAPE) {
+      stop()
+      this.textArea.current.blur()
+    }
+  }
```

Comment:
This functionally isn't changing anything but it's easier to when integrating the component with other components listening to escape to prevent actions bound to escape from firing based on the state of the component if we delay blurring the input to the keyup event.